meta/main.yml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
I don't think this file is necessary and should probably be removed. If it is required, at least all of the boilerplate should be stripped out.

Reply:
I don't think we are using the file at this time, but I would like to have it here in case we might need to use ansible-galaxy info for the role in the future, at least we have a default format of how to do it in the file